lib/components/term.js

```diff
@@ -37,7 +37,7 @@ export default class Term extends Component {
     prefs.set('cursor-color', this.validateColor(props.cursorColor, 'rgba(255,255,255,0.5)'));
     prefs.set('enable-clipboard-notice', false);
     prefs.set('foreground-color', props.foregroundColor);
-    prefs.set('background-color', 'transparent');
+    prefs.set('background-color', props.backgroundColor);
```

Comment:
maybe we should add a comment here about the expected override in `hterm.js` for this to work